### PR TITLE
pre-release steps for 1.0.0-beta1

### PR DESCRIPTION
**Pull request by kschoche**
Changes proposed in this PR:
- update chart values
How I've tested this PR:

How I expect reviewers to test this PR:

Checklist:
- [ ] Tests added
- [x] CHANGELOG entry added 
  > HashiCorp engineers only, community PRs should not add a changelog entry.
  > Entries should use present tense (e.g. Add support for...)

### CODE REVIEW

File: charts/consul/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ annotations:
     - name: consul
       image: hashicorp/consul:1.13.2
```
Comment (david-yu) [inline charts/consul/Chart.yaml:16]:
```suggestion
      image: hashicorp/consul:1.14.0-beta1
```

**Review by david-yu — APPROVED** [review]

File: charts/consul/values.yaml [hunk 2]
```diff
@@ -129,7 +129,7 @@ global:
   # image that is used for functionality such as catalog sync.
   # This can be overridden per component.
   # @default: hashicorp/consul-k8s-control-plane:<latest version>
-  imageK8S: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.0.0-dev
+  imageK8S: hashicorp/consul-k8s-control-plane:1.0.0-beta1
```
Comment (ishustava) [inline charts/consul/values.yaml:132]: Could we update the consul image as well?

Comment (jmurret) [reply charts/consul/values.yaml:132]: +1

**Review by jmurret — APPROVED** [review]
Would like to see the consul image helm value default updated as well, but not blocking.

File: charts/consul/Chart.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: consul
-version: 1.0.0-dev
+version: 1.0.0-beta1
 appVersion: 1.13.2
```
Comment (david-yu) [inline charts/consul/Chart.yaml:4]:
```suggestion
appVersion: 1.14.0-beta1
```

**Review by thisisnotashwin — APPROVED** [review]
We should also update test.yml under `.github/workflows` to use 1.14.0-beta1 instead of 1.13.1. Hitting approve prematurely in anticipation of the images getting their values updated as suggested by David and Iryna. Thanks so much for doing this Kyle

**Review by curtbushko — APPROVED** [review]

**Pull request merged**